### ISSUE TITLE
Support cover component

### ISSUE THREAD

**Issue by FreshlyBrewedCode** [FreshlyBrewedCode/hacs-homee#2]
**Is your feature request related to a problem? Please describe.**
Smart covers/shutters, gates, garages and similar devices are currently not supported. 

**Describe the solution you'd like**
Implement the homee platform for the Home Assistant [cover component](https://www.home-assistant.io/integrations/cover/).

**Describe alternatives you've considered**
Use the Home Assistant [template platform](https://www.home-assistant.io/integrations/#search/template) in combination with webhooks or switches to control unsupported devices. 

**Additional context**
Possible related [homee node profiles](https://github.com/FreshlyBrewedCode/pymee/blob/370006b58fb7a793ba130b8f5998b6c941c41bba/pymee/const.py#L1):
```python
ENTRANCE_GATE_OPERATOR = 2010
GARAGE_DOOR_OPERATOR = 2012
GATE_OPERATOR = 2013
INNER_DOOR_OPERATOR = 2014
GARAGE_DOOR_IMPULSE_OPERATOR = 2015
SHUTTER_POSITION_SWITCH = 2002
```

**Comment by EricReiche** [comment]
Hey,
I don't know Python unfortunately, but I got a Hörmann Garage door (Supramatic) and a yard gate (Rotamatic), can I support this development somehow?

**Comment by FreshlyBrewedCode** [comment]
> Hey,
> I don't know Python unfortunately, but I got a Hörmann Garage door (Supramatic) and a yard gate (Rotamatic), can I support this development somehow?

Thank you for your interest in the project. Sadly I currently don't have enough time to push the project forward and implement new features. Once I find time and get back to this feature I will make sure to reach out and gather feedback.

**Comment by HennigMatthias** [comment]
Hey,

is there any chance you plan to implement covers any time soon?

Greetings

**Comment by FreshlyBrewedCode** [comment]
> Hey,
> 
> is there any chance you plan to implement covers any time soon?
> 
> Greetings

Sorry, since I am not an active homee user anymore, I won't continue development on this project myself anymore.

**Comment by FreshlyBrewedCode** [comment]
Closed by #13. Released in v1.5.0
